websites/W/Warriorcats/presence.ts

```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+		clientId: "936630095035113522",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getPostNumber() {
+	return document
+		.querySelector("article")
+		.getAttribute("id")
+		.replace(/post-/, "");
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		details: "Stöbert auf der Warrior Cats Seite",
+		largeImageKey: "https://i.imgur.com/Ij2BuGr.jpg",
+		startTimestamp: browsingTimestamp,
+		buttons: [{ label: "Current Page", url: document.location.href }],
```

Comment:
This [button](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) could lead to a main page and is also very generic.
Rather than deleting buttons on paths that shouldn't have buttons, add specific buttons instead.